e2e/support/helpers/e2e-dashboard-helpers.js

```diff
@@ -248,6 +242,78 @@ export function createTextCard({
   };
 }
 
+export function createHeadingCard() {
```

Comment:
All changes outside of `e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js` are related to introduction of `createHeadingCard`, `createActionCard`, `createLinkCard` and removal (i.e. inlining) of old `createLinkCard`.